_data/rules.yml

```diff
@@ -301,6 +301,22 @@ groups:
           - name: prometheus/mysqld_exporter
             doc_url: https://github.com/prometheus/mysqld_exporter
             rules:
+              - name: MySQL down
+                description: MySQL down on {{ $labels.instance }}
+                query: 'mysql_up == 0'
+                severity: error            
+              - name: MySQL connections number is high
+                description: MySQL connections numbers approach the maximum number of connexion possible : more than 80% of connection are in use on {{ $labels.instance }}
+                query: '(mysql_global_status_threads_connected / mysql_global_variables_max_connections) * 100 > 80'
+                severity: error
+              - name: MySQL slow query count
+                description: MySQL server is having some slow query. The count is higher than 0 on {{ $labels.instance }}
+                query: 'rate(mysql_global_status_slow_queries[5m]) > 0'
```

Comment:
Could be replaced by: `query: 'mysql_global_status_slow_queries > 0'`

The 5min rate is equivalent to: `for: 5m` in alert configuration.

Reply:
Commit https://github.com/samber/awesome-prometheus-alerts/pull/95/commits/5559e0140b047c7afb3142307ea6442cb5de9b70 fix this.